src/views/GeneralConfig.vue

```diff
@@ -594,6 +663,7 @@ export default {
 				"openWB/general/extern",
 				"openWB/general/control_interval",
 				"openWB/general/grid_protection_configured",
+				"openWB/general/http_api",
```

Comment:
Wir sollten zwei Topics in diesem Zweig verwenden:
`openWB/general/http_api/https_enabled` ersetzt die bereits umgesetzte Einstellung und aktiviert lediglich den Zugriff über HTTPs.
Mit `openWB/general/http_api/http_enabled` kann explizit auch der unverschlüsselte Zugriff freigegeben werden.